topics/transcriptomics/tutorials/de-novo/tutorial.md

```diff
@@ -118,6 +118,9 @@ For quality control, we use similar tools as described in [NGS-QC tutorial]({{si
 >    > {: .solution }
 >    {: .question}
 > ![Before and after trimming comparison](../../images/BeforeAndAfterTrimming.png)
+>
+> 4. **Trimmomatic** {% icon tool %}: Run `Trimmomatic` on the remaining forward/reverse read pairs with the same parameters.

```

Comment:
Hmm why do you want to rerun trimmomatic?

Reply:
Because we ask them to `Run HISAT2 on the remaining forward/reverse read pairs with the same parameters.` so I feel that we should also run Trimmomatic, isn't?

Reply:
Ah yes, got it